.travis.yml

```diff
@@ -1,8 +1,13 @@
 language: python
 python: 2.7
+sudo: false
+cache:
+  directories:
+  - eggs
 
 env:
-  - PLONE_VERSION=4.3 QA=true
+- PLONE_VERSION=4.3
+- PLONE_VERSION=5.0
```

Comment:
Please don't do this yet :)
plone.api should work in Plone 5, but the tests really do not :)